scrapers/DDFNetwork.yml

```
@@ -15,12 +15,15 @@ xPathScrapers:
   sceneScraper:
     scene:
       Title: //meta[@itemprop="name"]/@content
-      Date: //meta[@itemprop="uploadDate"]/@content
+      Date:
+        selector: //div[@class='title-border d-inline-flex pr-3 flex-sm-fill justify-content-sm-center pr-sm-0']/p[@class='m-0 text-center']
```

if I may suggest this instead
```suggestion
        selector: //div[@id="video-specs"]//p[text()]
```

---

oh, wait, it fails on some sites